internal/util/exit.go

```diff
@@ -1,26 +1,18 @@
 package util
 
 import (
-	"os"
-
 	"github.com/edulinq/autograder/internal/log"
 )
 
-var (
-	lastExitCode         = 0
-	ShouldExitForTesting = true
-)
-
+// Everyone should use Exit() for coherent code.
```

Comment:
"Coherent code" is a bit abstract.
I think something more direct would be better, like:
"Everyone should use Exit() instead of exiting via something like os.Exit().
This allows for easier testing."